snow/engine/common/atomic_app_handler.go

```diff
@@ -1,36 +1,35 @@
 // Copyright (C) 2019-2023, Ava Labs, Inc. All rights reserved.
 // See the file LICENSE for licensing terms.
 
-package network
+package common
```

Comment:
The `common` package is already pretty bloated... not really sure I like this moving there (I'm hoping to remove this entire folder at some point)